scheduler/scheduler-server/src/test/java/functionaltests/api/TestLoadJobs.java

```diff
@@ -132,6 +132,7 @@ public void testLoadJobs() throws Exception {
         job = jobs.get(0);
 
         assertEquals(this.getClass().getSimpleName(), job.getJobId().getReadableName());
+        assertEquals(this.getClass().getSimpleName(), job.getProjectName());
```

Comment:
This means that `job.getJobId().getReadableName()` has the same value as `job.getProjectName()` in this test?